lib/cdo/redshift_import.rb

```diff
@@ -0,0 +1,54 @@
+require 'cdo/redshift'
+
+class RedshiftImport
+  # Database Migration Service Replication Tasks load data from Aurora into staging Redshift tables with a prefix.
+  TEMP_TABLE_PREFIX = '_import_'.freeze
+
+  # DMS Replication Tasks import data into temporary / staging tables prefixed with "_import_".
+  # Complete import of staged data from production database into Redshift by truncating each
+  # target table, moving all of the rows from each import table to its corresponding target table, and then dropping
+  # each temporary / staging import table.
+  # @param schemas [String] Array of Redshift schemas in which to complete data import.
+  def self.complete_table_import(schemas)
+    schemas.each do |schema|
+      temporary_import_tables(schema).each do |import_table|
+        target_table = import_table.partition(TEMP_TABLE_PREFIX).last
+        truncate_table(schema, target_table)
+        move_rows(schema, import_table, schema, target_table)
+        drop_table(schema, import_table)
+      end
+    end
+  end
+
+  # Get names of tables in a Redshift schema that were loaded via DMS to stage imported data from the database.
+  # @param schema [String] The Redshift schema to search for database import staging tables.
+  # Returns Array of Redshift table names.
+  def self.temporary_import_tables(schema)
+    redshift_client = RedshiftClient.instance
+    query = <<-SQL
+SET search_path TO #{schema};
```

Comment:
Formatting nit: learned about this recently in case it's of interest:

https://infinum.com/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc

Reply:
That's much better!  I've updated to use squiggly HEREDOC.